reducers/sessionReducer.js

```diff
@@ -31,6 +37,9 @@ export default (state, action) => {
       return state;
     case UPDATE_CURRENT_USER:
       return Object.assign({}, state, { user: action.user });
+    case USER_ALREADY_LOGGED_IN:
+      // todo: hide everything, display message
```

Comment:
Something like the below image, but I don't know much about UI stuff.

![image](https://user-images.githubusercontent.com/13383097/53778910-5ef04080-3f39-11e9-8816-bb1c7f62f156.png)

Reply:
IMHO a better solution would allow the user to control the player from another tab, but reuse the same refresh token. Even better, it should be possible to just remove duplicates from the list of users.

I can imagine some users will have many tabs on several windows/browsers. Also, it might be that the are logged in on another device. It might be more user friendly to let them use the site and handle the duplicates in the app, instead of moving the burden to the user.

Reply:
Yes, you're right. But maybe we should control some actions like "play track". Imagine that a user have multiple tabs, every tab will receive a "play track" message. But it will bring a lot of complexity in order to control this behavior. What do you think? Is multiple "play track" messages acceptable?

Reply:
I see your point. There would be multiple tabs sending play commands, which might not get them exactly at the same time. The backend could be smart enough to only send it to one of the users from the duplicated set, though I don't know if this is overcomplicating things.

In general I prefer to put logic in the server rather than making the user think and find out what's going on.

What do you think?

Reply:
Yes, it's better to do these things in server side. So one solution is to retrofit `user`, it has a `sockets` attributes which contains an array of sockets associated with the user. When sending `play track` command, don't simply use `broadcast`, instead, iterate `users`, for each user, pick one of those sockets to send `play track` command.

Is this a good solution?

Reply:
Yes, that sounds great!

Reply:
solve this problem in a new commit. Hope that it's good to go. 😂